timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -109,6 +135,7 @@ private void registerLeaderElectionService(TimeLockServerConfiguration configura
                 localPaxosServices.ourAcceptor(),
                 localPaxosServices.ourLearner()));
         environment.jersey().register(new NotCurrentLeaderExceptionMapper());
+        environment.jersey().register(new TooManyRequestsExceptionMapper());
```

Comment:
Hmm, or this bit could go with the rate limiting configuration, as it's relevant to that and not the leader election service.